pkg/security/admission/scc_exec.go

```diff
@@ -53,7 +54,7 @@ func (d *sccExecRestrictions) Admit(a admission.Attributes) (err error) {
 	// we're allowed to use the SA the pod is using.  Otherwise, user-A creates pod and user-B (who can't use the SA) can exec into it.
 	createAttributes := admission.NewAttributesRecord(pod, nil, kapi.Kind("Pod").WithVersion(""), a.GetNamespace(), a.GetName(), a.GetResource(), "", admission.Create, a.GetUserInfo())
 	if err := d.constraintAdmission.Admit(createAttributes); err != nil {
-		return admission.NewForbidden(a, fmt.Errorf("%s operation is not allowed because the pod's security context exceeds your permissions: %v", a.GetSubresource(), err))
```

Comment:
> you should fix the upstream method to give a reasonable result

Here is the upstream method: https://github.com/openshift/origin/blob/3bb18b610302474658f1fa55e0f68c31f9bbf2d5/pkg/security/admission/admission.go#L134

It invokes `admission.NewForbidden()` and looks like it's correct. As I mention in the description, for some unknown reason, we deal with an error of the different type. I couldn't find who/how has modified the error between these admission controllers.

Ok, tomorrow I'll look at it again. Maybe you have some ideas on that?